Sources/HiPMobile/BusinessLayer/ContentApiFetchers/BaseDataFetcher.cs

```diff
@@ -87,12 +86,16 @@ public BaseDataFetcher(IRoutesBaseDataFetcher routesBaseDataFetcher, IExhibitsBa
                     transaction.Rollback();
                     return;
                 }
-                dataToRemoveFetcher.CleaupRemovedData();
                 if (token.IsCancellationRequested)
                 {
                     transaction.Rollback();
                 }
             }
-        }
+            await dataToRemoveFetcher.FetchDataToDelete (token);
```

Comment:
Indentation looks wrong here, can you go through the files and hit CTRL-K and CTRL-D consecutively to auto format?